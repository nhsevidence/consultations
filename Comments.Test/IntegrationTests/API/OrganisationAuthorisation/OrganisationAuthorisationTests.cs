using Comments.Test.Infrastructure;
using System.Threading.Tasks;
using Xunit;

namespace Comments.Test.IntegrationTests.API.OrganisationAuthorisation
{
	public class OrganisationAuthorisationTests : TestBase
	{
		[Fact]
		public async Task GenerateCode()
		{
			//Arrange (in the base constructor for this one.)

			// Act
			var response = await _client.PostAsync("/consultations/api/OrganisationAuthorisation?organisationId=1&consultationId=1", null);
			response.EnsureSuccessStatusCode();
			var responseString = await response.Content.ReadAsStringAsync();

			// Assert
			responseString.ShouldMatchApproved();
		}

	}
}
