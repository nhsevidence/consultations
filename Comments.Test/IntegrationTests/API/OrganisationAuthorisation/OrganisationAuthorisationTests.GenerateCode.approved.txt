{
    "organisationAuthorisationId":"scrubbed by ScrubUserId",
    "organisationId":1,
    "organisationName":"NICE",
    "canGenerateCollationCode":false,
    "collationCode":"1234 1234 1234"
}